docs/views/about.html

```diff
@@ -24,21 +24,20 @@ <h1 class="govuk-heading-xl">About</h1>
       <p class="govuk-body-l">
         The GOV.UK Prototype Kit provides a simple way to make interactive prototypes that look like pages on GOV.UK. These prototypes can be used to show ideas to people you work with, and to do user research
       </p>
-      
+
       <h2 class="govuk-heading-m">Support</h2>
-      
+
       <p>The GOV.UK Prototype Kit is maintained by the Government Digital Service. If you’ve got a question or need support you can:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>email 
+        <li>email
           <a href="mailto:govuk-design-system-support@digital.cabinet-office.gov.uk" class="govuk-link">
             govuk-design-system-support@digital.cabinet-office.gov.uk
           </a>
         </li>
-        <li>
-          <a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit" class="govuk-link">
-            get in touch on Slack
-          </a> 
-          (<a class="govuk-link" href="slack://channel?team=T04V6EBTR&amp;id=C0647LW4R">open in app</a>)
+        <li>get in touch on Slack <a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit" class="govuk-link">
+          (open Prototype Kit channel in desktop)
```

Comment:
Should we add a forward slash (or the word 'or') between 'open Prototype Kit channel in desktop' and 'open Prototype Kit channel in app'?